.gitignore

```diff
@@ -1,2 +1,3 @@
 /CMakeLists.txt
 .catkin_tools/
+.idea/
```

Comment:
For local folders like this we'd recommend using a global gitignore file as described e.g. [here](https://stackoverflow.com/questions/7335420/global-git-ignore).